man/ostree.repo-config.xml

```diff
@@ -87,20 +87,18 @@ Boston, MA 02111-1307, USA.
       </varlistentry>
 
       <varlistentry>
-        <term><varname>commit-update-summary</varname></term>
+        <term><varname>auto-update-summary</varname></term>
```

Comment:
I like the name `auto-update-summary`, but it also makes it sound like any change that would affect the summary triggers a summary update, which isn't quite true. Maybe we should mention that some operations such as generating a static delta or setting `core.collection-id` in the repo config would make the summary stale but not trigger an update with this option set?